rasa/core/actions/two_stage_fallback.py

```diff
@@ -85,7 +85,7 @@ async def is_done(
     ) -> bool:
         _user_clarified = _last_intent_name(tracker) not in [
             DEFAULT_NLU_FALLBACK_INTENT_NAME,
-            USER_INTENT_OUT_OF_SCOPE,
+            USER_INTENT_REJECT_SUGGESTION,
```

Comment:
@ricwo The problem I see with this approach is that it potentially breaks existing assistants which are currently using `out_of_scope`. In that sense it would be better to keep the old intent. I think remembering which intents were added as "default intents" and which were added by the users might make the most sense then?

Reply:
👍  let's switch to the other approach - allowing overriding default intents